docs/api/primitives.md

```diff
@@ -42,7 +46,7 @@ String value of the error type. Could be `ARROWHEAD`, `INVALID_PARAMETER`, `AUTH
 
 ## InterfaceName
 
-A String identifier that is intended to be both human and machine-readable. The allowed characters are letters (english alphabet only) and numbers. A name has to start with a letter and must following the snake_case naming convention. The identifier maximum length is 63 characters.
+A String identifier that is intended to be both human and machine-readable. The allowed characters are letters (english alphabet only), numbers and underscore (_). A name has to start with a letter and must following the snake_case naming convention. The identifier maximum length is 63 characters.
```

Comment:
please change must following to must follow in this file and your sds as well. I made this mistake and now it is everywhere via copy-pasting.